server/transport.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/go-kit/kit/endpoint"
 	kitlog "github.com/go-kit/kit/log"
 	kithttp "github.com/go-kit/kit/transport/http"
-	"golang.org/x/net/context"
+	"context"
```

Comment:
I'm not sure how familiar you are with Go, but the dependency management is not great. There's lots of tools `glide, dep, godep, govendor` etc which are all trying to manage versions.